### ISSUE TITLE
Gives error cannot read property 'workspaceFolders' of undefined

### ISSUE THREAD

**Issue by farzadmf** [josa42/coc-go#90]
Hi, I'm getting the following error:
```
2020-07-10T11:34:08.217 ERROR (pid:509699) [server] - unhandledRejection  Promise {
  <rejected> TypeError: Cannot read property 'workspaceFolders' of undefined
      at registerGopls (/home/farzad/.config/coc/extensions/node_modules/coc-go/lib/extension.js:44:49)
} TypeError: Cannot read property 'workspaceFolders' of undefined
    at registerGopls (/home/farzad/.config/coc/extensions/node_modules/coc-go/lib/extension.js:44:49)
```
I _think_ it's because of change recently added that tries to read `config.disable.workspaceFolders` [here](https://github.com/josa42/coc-go/blob/master/src/extension.ts#L55) (I _think_)

**Comment by FedericoAntoniazzi** [comment]
I had the same problem but I didn't add config.disable.workspaceFolders.
### What was the error
A red bar appeared in the bottom saying 
`[coc.nvim]: UnhandledRejection: TypeError: Cannot read property 'workspaceFolders' of undefined`.
The strangest thing was that I had no problems in python projects
### How I solved the problem
I've only updated Plug and coc-nvim extensions
```
:PlugUpdate
:CocUpdate
```

Hope you can find this useful :)

**Comment by dakyskye** [comment]
> I had the same problem but I didn't add config.disable.workspaceFolders.
> ### What was the error
> 
> A red bar appeared in the bottom saying
> `[coc.nvim]: UnhandledRejection: TypeError: Cannot read property 'workspaceFolders' of undefined`.
> ### How I solved the problem
> 
> I've only updated Plug and coc-nvim extensions
> 
> ```
> :PlugUpdate
> :CocUpdate
> ```
> 
> Hope you can find this useful :)

I just did your steps, but I am getting the same red error bar with the same error.

**Comment by dakyskye** [comment]
Oh sorry I'm wrong, now I get `Cannot read property 'diagnostics' of undefined`

**Comment by pascalharp** [comment]
I had the same issue. #89 fixed it for me.

**Comment by FedericoAntoniazzi** [comment]
> I had the same problem but I didn't add config.disable.workspaceFolders.
> ### What was the error
> A red bar appeared in the bottom saying 
> `[coc.nvim]: UnhandledRejection: TypeError: Cannot read property 'workspaceFolders' of undefined`.
> 
> ### How I solved the problem
> I've only updated Plug and coc-nvim extensions
> ```
> :PlugUpdate
> :CocUpdate
> ```
> 
> Hope you can find this useful :)

Nevermind, the error reappeared :(
Luckily, solutions given in the thread are working fine, thank you

**Comment by josa42** [comment]
Fixed in `0.9.2`
